addons/account/models/account_move.py

```diff
@@ -614,6 +614,16 @@ def _sequence_year_range_monthly_regex(self):
         tracking=True,
         help="It indicates that the invoice/payment has been sent or the PDF has been generated.",
     )
+
+    is_move_sent_label = fields.Selection(
+        selection=[
+            ("sent", "Sent"),
+            ("not_sent","Not sent")
+        ],
+        string='Invoice Sent',
+        compute='_compute_is_move_sent_label'
```

Comment:
will not block for that but rule of thumb here, double quotes for what the user will see and single otherwise
```suggestion
        selection=[
            ('sent', "Sent"),
            ('not_sent',"Not sent")
        ],
        string="Invoice Sent",
        compute='_compute_is_move_sent_label'
```